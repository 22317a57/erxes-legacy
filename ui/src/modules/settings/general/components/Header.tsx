import HeaderDescription from 'modules/common/components/HeaderDescription';
import React from 'react';

function Header() {
  return (
    <HeaderDescription
			icon="/images/actions/25.svg"
			title="System Config"
			description="Set up your initial account settings so that things run smoothly in unison."
		/>
  );
}

export default Header;