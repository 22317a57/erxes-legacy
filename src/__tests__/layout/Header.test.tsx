import { mount, shallow } from 'enzyme';
import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Header from '../../modules/layout/components/Header';

describe('Header component', () => {
  const defaultProps = {
    title: 'default'
  };

  test('renders Header successfully', () => {
    shallow(<Header {...defaultProps} />);
  });

  test('renders successfully with default value', () => {
    const wrapper = mount(<Header {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });

  test('snapshot matches', () => {
    const rendered = renderer.create(<Header {...defaultProps} />).toJSON();

    expect(rendered).toMatchSnapshot();
  });
});
