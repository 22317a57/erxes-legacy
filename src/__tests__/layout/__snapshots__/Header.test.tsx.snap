// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapshot matches 1`] = `
<div
  className="sc-keFjpB ibRdUp"
/>
`;
